addons/account_analytic_analysis/account_analytic_analysis.py

```diff
@@ -702,6 +702,7 @@ def _prepare_invoice_data(self, cr, uid, contract, context=None):
            'fiscal_position': fpos and fpos.id,
            'payment_term': partner_payment_term,
            'company_id': contract.company_id.id or False,
+           'user_id': contract.manager_id.id,
```

Comment:
If we have no account manager on a contract, we would get no longer a saleman. Maybe adding `or uid` ?